modules/vertx-swagger-codegen/src/main/resources/javaVertXServer/apiVerticle.mustache

```diff
@@ -33,11 +33,22 @@ public class {{classname}}Verticle extends AbstractVerticle {
                 {{/isString}}{{^isString}}{{{dataType}}} {{paramName}} = Json.mapper.readValue(message.body().getString("{{baseName}}"), {{{dataType}}}.class);
                 {{/isString}}{{/isPrimitiveType}}{{^isPrimitiveType}}{{{dataType}}} {{paramName}} = Json.mapper.readValue(message.body().getJsonObject("{{baseName}}").encode(), {{{dataType}}}.class);
                 {{/isPrimitiveType}}{{/isListContainer}}{{/allParams}}{{/hasParams}}
-                {{#returnType}}{{{returnType}}} result = service.{{operationId}}({{#hasParams}}{{#allParams}}{{paramName}}{{#hasMore}}, {{/hasMore}}{{/allParams}}{{/hasParams}});
-                {{#isListContainer}}message.reply(new JsonArray(Json.encode(result)).encodePrettily());{{/isListContainer}}{{^isListContainer}}message.reply(new JsonObject(Json.encode(result)).encodePrettily());{{/isListContainer}}{{/returnType}}{{^returnType}}service.{{operationId}}({{#hasParams}}{{#allParams}}{{paramName}}{{#hasMore}}, {{/hasMore}}{{/allParams}}{{/hasParams}});
-                message.reply(null);{{/returnType}}
```

Comment:
Good find, thanks! Was not critical, but it lead to a "null" string being returned in the HTTP response for operations without result, not nice. I fixed that in the latest commit, please check.

Btw, I am new to mustache, is there a way to format/indent the template without formatting/indenting the output? That would be awesome.